orttraining/orttraining/python/ort_trainer.py

```diff
@@ -902,23 +915,26 @@ def _debug_conversion_step(self, input, input_descs, output_desc):
         cpu_data = tuple(cpu_data_list)
 
         # torch inference step
-        with torch.no_grad(): torch_out = [self.torch_model_(*cpu_data)]
+        with torch.no_grad():
+            torch_out = [self.torch_model_(*cpu_data)]
         
         # iterate through outputs, except loss when loss_fn is distinct from the model
-        if self.loss_fn_: torch_out.insert(0, torch.empty(1, 1))
+        if self.loss_fn_:
+            torch_out.insert(0, torch.empty(1, 1))
         for ((out_name, npp_out), pp_out, t_out) in zip(no_postprocess_session_results.items(), postprocess_session_results.values(), torch_out):
-            if out_name == 'loss' and self.loss_fn_: continue
+            if out_name == 'loss' and self.loss_fn_:
```

Comment:
```suggestion
            if out_name == self.model_desc_.outputs[0].name and self.loss_fn_:
```

Reply:
as the loss name is provided by the user and they are free to type whatever they want, we shouldn't assume 'loss' as the loss name.

What is known is that the first output is always loss, so we can use that